imports/api/accounts/server/methods.js

```diff
@@ -82,8 +82,11 @@ Meteor.methods({
         try{
             let rewards = HTTP.get(url);
             if (rewards.statusCode == 200){
-                balance.rewards = JSON.parse(rewards.content).result.total;
-                console.log("I AM CALLIJG YOU JE");
+                //get seperate rewards value
+                balance.rewards = JSON.parse(rewards.content).result.rewards;
+                //get total rewards value
+                balance.total_rewards= JSON.parse(rewards.content).result.total;
```

Comment:
Variables should be in `camelCase`.

https://github.com/forbole/big_dipper/blob/master/CONTRIBUTING.md#create-pull-request